### ISSUE TITLE
paper？

### ISSUE THREAD

**Issue by wusiqi521** [ykdai/Flare7K#1]
Have you applied the uformer network to this field without trying to publish a paper?

**Comment by ykdai** [comment]
We have published a paper 'Flare7K: A Phenomenological Nighttime Flare Removal Dataset' and it was accepted by the NeurIPS 2022 Track Datasets and Benchmarks. Uformer is the baseline method of this paper. 
You can check our paper at: https://openreview.net/pdf?id=Proso5bUa

**Comment by wusiqi521** [comment]
> 

Thank you for your reply, I would also like to ask you if the loss function you used in uformer network training is based on the one when Wu et al. (unet)?

**Comment by ykdai** [comment]
Yes, we follow Wu et al.'s loss function's settings and use L1 and perceptual loss to train our networks.

**Comment by wusiqi521** [comment]
thanks！
